register/templates/register/forgot_password.html

```diff
@@ -0,0 +1,135 @@
+    {{user_form.errors}}
+      <strong>{{ error|escape }}</strong>
+  </div>
+  {% endif %}
+  {% endfor %} -->
+  
+  <div class="registerForm">
+    <div class="row text-center">
+      <div class="col-12">
+        <h2>Forgot Password?</h2>
+        <p>Enter your username below</p>
+        <!-- <p>Password must be alteast 8 characters long</p> -->
+      </div>
+    </div>
+    <hr class="hrLine" />
+    <!-- {% for field in user_form %}
+    <div class="row form-group justify-content-center">
+      <div class="col-12 col-md-4">
+        <label for="email"> {{ field.label }} </label>
+        {{ field }}
+      </div>
+    </div>
+    {% endfor %} -->
+    {% csrf_token %}
+    <div class="row form-group justify-content-center"> 
+        <!-- {{user_form|crispy}} -->
+        <div class="col-12 col-md-4">
+          <div class="form-group">
+            <strong><label for="username" style="font-size: 1.25rem;">Username<strong style="color:red;">*</strong></label></strong>
+            <div> <input type="text" id="kk" name="username" required/></div>
+          </div>
+          <!-- <div class="form-group">
+            <strong><label for="fname" style="font-size: 1.25rem;">First Name<strong style="color:red;">*</strong></label></strong>
+            <div> <input type="text" id="kk" name="fname"  required/></div>
+          </div>
+          <div class="form-group">
+            <strong><label for="lname" style="font-size: 1.25rem;">Last Name<strong style="color:red;">*</strong></label></strong>
+            <div> <input type="text" id="kk" name="lname" required/></div>
+          </div>
+          <div class="form-group">
+            <strong><label for="email" style="font-size: 1.25rem;">Email ID<strong style="color:red;">*</strong></label></strong>
+            <div> <input type="email" id="kk" name="email" required/></div>
+          </div> -->
+          <div class="form-group">
+            <strong><label for="password1" style="font-size: 1.25rem;">New Password<strong style="color:red;">*</strong></label></strong>
+            <div> <input type="password" id="kk" name="password1" required/></div>
+            <div style="margin-top:0.5rem"><strong style="color:red;">*</strong>&nbsp;Password must be atleast 8 characters long</div>
+          </div>
+          <div class="form-group" style="font-size: 1.25rem;">
+          <strong><label for="password2">Confirm Password<strong style="color:red;">*</strong></label></strong>
+            <div> <input type="password" id="kk" name="password2"required/></div>
+          </div>
+        </div>
+    </div>
+    <div class="row form-group justify-content-center">
+      <div class="col-12 col-md-4">
+        <input type="submit" class="btn btn-primary btn-block" name="submit" value="Reset Password">
+      </div>
+    </div>
+  </div>
+</form>
+<script type="text/javascript" src="{% static 'js/update_theme_register.js' %}"></script>
+{% endblock %}
+
+
+
+
+
+
+
+
+
+
+{%comment%}
```

Comment:
this is commented code, right? please all the commented code if it's not relevant